src/connection.rs

```diff
@@ -432,6 +432,15 @@ impl<T: AsyncRead + AsyncWrite + Unpin> Sender<T> {
         self.send_frame(&mut header, &mut Storage::Shared(data.as_ref().as_bytes())).await
     }
 
+    /// Send a text value over the websocket connection.
+    ///
+    /// In contrast to [`Sender::send_text`] the provided data is modified
+    /// in-place, e.g. if masking is necessary.
```

Comment:
I think what @dvdplm said about this makes sense; perhaps tweak the comment to just note that this is just a potential optimisation if owned data can be provided